Documentation/building/osx-instructions.md

```diff
@@ -15,23 +15,13 @@ Git Setup
 
 Clone the CoreCLR and CoreFX repositories (either upstream or a fork).
 
-    dotnet-mbp:git richlander$ git clone https://github.com/dotnet/coreclr
-    Cloning into 'coreclr'...
-    remote: Counting objects: 16526, done.
-    remote: Compressing objects: 100% (21/21), done.
-    remote: Total 16526 (delta 8), reused 0 (delta 0), pack-reused 16505
-    Receiving objects: 100% (16526/16526), 26.26 MiB | 9.87 MiB/s, done.
-    Resolving deltas: 100% (7679/7679), done.
-    Checking connectivity... done.
-
-    dotnet-mbp:git richlander$ git clone https://github.com/dotnet/corefx
-    Cloning into 'corefx'...
-    remote: Counting objects: 41610, done.
-    remote: Compressing objects: 100% (14/14), done.
-    remote: Total 41610 (delta 5), reused 0 (delta 0), pack-reused 41596
-    Receiving objects: 100% (41610/41610), 23.97 MiB | 311.00 KiB/s, done.
-    Resolving deltas: 100% (25680/25680), done.
-    Checking connectivity... done
+```sh
+dotnet-mbp:git richlander$ git clone https://github.com/dotnet/coreclr
+# Cloning into 'coreclr'...
+
+dotnet-mbp:git richlander$ git clone https://github.com/dotnet/corefx
+# Cloning into 'corefx'...
+```
 
 This guide assumes that you've cloned the coreclr and corefx repositories into `~/git/coreclr` and `~/git/corefx` on your OS X machine and the coreclr repository into `C:\git\coreclr` on Windows. If your setup is different, you'll need to pay careful attention to the commands you run. In this guide, I'll always show you the current directory on both the OS X and Windows machine.
```

Comment:
Good idea James! I have removed all occurrences of "Windows" from OS X instructions! This is like dream comes true that we need just one OS, any OS, to build the whole thing end-to-end. PR #4953 is a big and v.important step towards that direction, especially if you are someone trying to build coreclr often on different kinds of operating systems and are forced to `scp` 2.8 GBs large `bin/tests` dir from Windows to target. The worst part is that `tests\buildtest.cmd clean` on Windows always run into (couple of hundreds of) errors like:

> CSC : error CS0006: Metadata file 'C:\Users\Peter\Source\Repos\coreclr\tests\\..\bin\Product\Windows_NT.x64.Debug\ref\m
scorlib.dll' could not be found [C:\Users\Peter\Source\Repos\coreclr\tests\src\JIT\Methodical\xxobj\operand\_speed_dbgr
efanyval.csproj]

(at which point I ignore these errors and scp to "foreign" OS anyway .. )